backend/app/mcp_server/tools/data_analysis.py

```
@@ -0,0 +1,250 @@
+            "attachment_id": attachment_id,
+            "tables": tables_schema,
+            "summary": info["summary"],
+            "content_ref": info["content_ref"],
+        }
+    except Exception as e:
+        logger.exception(
+            f"Error getting data schema for attachment {attachment_id}: {e}"
+        )
+        return {
+            "success": False,
+            "error": f"Failed to get schema: {str(e)}",
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)